src/main/java/org/carlmontrobotics/commands/RotateToFieldRelativeAngle.java

```diff
@@ -0,0 +1,48 @@
+package org.carlmontrobotics.commands;
+
+import org.carlmontrobotics.subsystems.Drivetrain;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.util.sendable.SendableRegistry;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import static org.carlmontrobotics.Constants.Drivetrain.*;
+
+public class RotateToFieldRelativeAngle extends Command {
+
+    public final TeleopDrive teleopDrive;
+    public final Drivetrain drivetrain;
+
+    public final PIDController rotationPID = new PIDController(thetaPIDController[0], thetaPIDController[1], thetaPIDController[2]);
+
+    public RotateToFieldRelativeAngle(Rotation2d angle, Drivetrain drivetrain) {
+        this.drivetrain = drivetrain;
+        this.teleopDrive = (TeleopDrive) drivetrain.getDefaultCommand();
+
+        rotationPID.enableContinuousInput(-180, 180);
+        rotationPID.setSetpoint(MathUtil.inputModulus(angle.getDegrees(), -180, 180));
+        rotationPID.setTolerance(positionTolerance[2], velocityTolerance[2]);
+        SendableRegistry.addChild(this, rotationPID);
```

Comment:
out of curiosity what is this line used for?

Reply:
This command is from last year but I assume line 27 makes rotationPID a sendable so that we can see it?